tfjs-backend-webgpu/src/unary_op_util.ts

```diff
@@ -166,6 +167,15 @@ const RELU_VEC4 = `
 `;
 const ROUND = `return round(a);`;
 const RSQRT = `return inverseSqrt(a);`;
+const SELU = `
+  // Stable and Attracting Fixed Point (0, 1) for Normalized Weights.
+  // see: https://arxiv.org/abs/1706.02515
```

Comment:
For the record, per discussion, we will try to keep the comments out of WGSL code in the future.